docs/graphql/core/migrations/advanced/resetting-migrations.rst

```diff
@@ -0,0 +1,63 @@
+.. meta::
+   :description: Resetting Hasura migrations
+   :keywords: hasura, docs, migration, reset migrations, clear migrations
+
+.. _reset_migration:
+
+Reset Hasura migrations
+==============================
```

Comment:
Underline should be the same length than the title.

Reply:
Great